docs/gettingstarted.rst

```diff
@@ -0,0 +1,141 @@
+Requirements
```

Comment:
`gettingstarted.rst` is introduced by converting README.md to .rst manually. Not sure this is the best way to manage two documents, though.

Reply:
This way, README and documentation site may become inconsistent and possibly cause confusion. Don't you directly replace `README.md` with `README.rst` and embed it to the `gettingstarted` page as follows? 

```
.. include:: ../README.rst
```

Reply:
I know that option, or we can use m2r to include markdown.
anyway, this is a Python project, I'll change README into rst format.

Reply:
Converted to rst format f49d516

Reply:
@chezou long_description in setup.py should be change to README.rst
https://github.com/treasure-data/td-client-python/blob/f49d5163a0654875310602c676ef2024b3d32a88/setup.py#L36

Reply:
@aodag Good catch! 4b7d687